cmake/OpenCVFindLibsVideo.cmake

```diff
@@ -1,320 +1,3 @@
-# ----------------------------------------------------------------------------
-#  Detect 3rd-party video IO libraries
-# ----------------------------------------------------------------------------
```

Comment:
We don't need this file anymore? Does this limit the `videoio` module's functionality?

Reply:
Here is a pointer to the upstream commit where this script was more or less removed https://github.com/opencv/opencv/commit/73959fed4581556d8e6b770987c819adfacceb87#diff-147a2d6c0a28c306ecf5078d84c5eb84 

I fixed my intermediate commits to now be in sync with the upstream file:
https://github.com/opencv/opencv/blob/4.1.1/cmake/OpenCVFindLibsVideo.cmake